lib/create-matcher.js

```diff
@@ -108,8 +108,8 @@ createMatcher.typeOf = function(type) {
 
 createMatcher.instanceOf = function(type) {
     if (
-        typeof Symbol === "undefined" ||
-        typeof Symbol.hasInstance === "undefined"
+        typeOf(Symbol) === "undefined" ||
```

Comment:
Isn't this causing `ReferenceError: Symbol is not defined` on platforms that don't support Symbol? I think the `typeof Symbol` check is needed for exactly that case, no?

Reply:
grrr, good question 😄 Not sure on this one honestly

Reply:
Seems you're right https://github.com/sinonjs/samsam/pull/118#issuecomment-538664573